src/main/python/main.py

```diff
@@ -0,0 +1,194 @@
+from fbs_runtime.application_context.PyQt5 import ApplicationContext
+import PyQt5.QtWidgets as QtWidgets
+import PyQt5.QtGui as QtGui
+import os
+import sys
+import resources  # noqa: F401
+
+sys.path.append(".")
+from lib.settings import load_settings, SETTINGS  # noqa: E402
+
+from model import Model  # noqa: E402
+from design_view import DesignView  # noqa: E402
+from file_view import FileView  # noqa: E402
+from play_view import PlayView  # noqa: E402
+from settings_view import SettingsView  # noqa: E402
```

Comment:
Maybe move these imports above `sys.path.append(".")`, so that we can remove all the `# noqa: E402` comments

Reply:
Then we need `sys.path.append(".")` in all these files as well.